test cases/rust/1 basic/meson.build

```diff
@@ -1,6 +1,6 @@
 project('rustprog', 'rust')
 
-e = executable('program', 'prog.rs',
+e = executable('rust-program', 'prog.rs',
```

Comment:
I haven’t looked for sure, but you probably need to update the test.json’s installed files list